frontend/src/components/BakerDetails/BakerDetailsPoolContent.vue

```diff
@@ -7,7 +7,7 @@
 					<template #title>Total stake</template>
 					<template #default>
 						<Tooltip
-							:text="`${formatPercentage(
+							:text="`${formatNumbers(
 								baker.state.pool.totalStakePercentage
 							)}% of all CCD in existence`"
```

Comment:
As mentioned [here](https://github.com/Concordium/concordium-scan/pull/840#issuecomment-3566955374), this is the percentage of all staked CCD and not the all CCD in existence.